lib/modules/job/widgets/main.widget.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:project/modules/job/widgets/jobCard.widget.dart';
+
+class JobMainWidget extends StatefulWidget {
+  @override
+  JobMainState createState() {
+    return JobMainState();
+  }
+
+}
+
+class JobMainState extends State<JobMainWidget> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text("Jobs"),
+      ),
+      body: JobCardWidget(),
+    );
+  }
```

Comment:
In general doar ecranele ar trebui sa aiba ca root un scaffold. Asta cum e widget normal ar trebui sa tina o lista de card-uri sau ceva similar.
In felul in care e implementat acum ai:
`JobMainScreen -> Scaffold -> Center -> JobMain -> Scaffold -> JobCardWidget -> ...`
Nu are logica sa ai un scaffold definit ca si copil al altui scaffold.
Eu as schimba in ceva de genul:
`JobMainScreen -> Scaffold -> JobMain -> ListView/Column/etc... -> [JobCard] -> ...`